docs/streams.html

```diff
@@ -790,13 +790,15 @@
         </p>
         <br>
 
-        <h2><a id="streams_upgrade" href="#streams_upgrade">Upgrade Guide and API Changes</a></h2>
+        <h2><a id="streams_upgrade_and_api_0102" href="#streams_upgrade_and_api_0102">Upgrade Guide and API Changes</a></h2>
```

Comment:
Makes sense. But still I'm thinking if it is necessary to change the hashtag on every release. I.e. on 0.10.2 release you would get a link of `/0102/documentation/streams#streams_upgrade_and_api_0102` and then on 0.10.3. you get `/0103/documentation/streams#streams_upgrade_and_api_0103`. Instead we can just name the hashtag as `#streams_upgrade_and_api` so that would be render for all versions and SEO can benefit from it as well.